lib/openapi3.js

```diff
@@ -44,18 +44,18 @@ function convertToToc(source,data) {
                     sMethodUniqueName = method.operation.summary;
                 }
                 method.slug = sMethodUniqueName.toLowerCase().split(' ').join('-'); // TODO {, } and : ?
-                var tagName = data.translations.defaultTag;
-                var tagDescription = '';
-                if (method.operation.tags && method.operation.tags.length > 0) {
-                    var tagData = getTagGroup(method.operation.tags[0], data.options.tagGroups);
-                    tagName = tagData.name;
-                    tagDescription = tagData.description;
+                if (typeof method.operation.tags === 'undefined' || !method.operation.tags.length) {
```

Comment:
`method.operation.tags` was left undefined in the original code. Meaning that you are not only allowing a method to be attached to several tags, but also making all methods associated with at least one tag, the default?

Reply:
Yup, but I don't care. 😎  On a more serious note though, this code adds method to the resources with the defaultTag anyway if no tags were defined, so it actually makes sense to have the tag added to the method itself too. Not that it's used for anything later on. Which is to say, personally I'm fine with this behaviour change, but might still yield to maintainer's will.

Reply:
Ok.